pom.xml

```diff
@@ -589,8 +589,8 @@
         <version.org.wildfly.extras.creaper>1.6.2</version.org.wildfly.extras.creaper>
         <version.org.wildfly.http-client>1.1.13.Final</version.org.wildfly.http-client>
         <version.org.wildfly.naming-client>1.0.15.Final</version.org.wildfly.naming-client>
-        <legacy.version.org.wildfly.security.elytron>1.19.0.Final</legacy.version.org.wildfly.security.elytron>
-        <version.org.wildfly.security.elytron>2.0.0.Beta2</version.org.wildfly.security.elytron>
+        <legacy.version.org.wildfly.security.elytron>1.20.1.Final</legacy.version.org.wildfly.security.elytron>
+        <version.org.wildfly.security.elytron>2.0.0.Beta3</version.org.wildfly.security.elytron>
         <legacy.version.org.wildfly.security.elytron-mp>1.19.0.Final</legacy.version.org.wildfly.security.elytron-mp>
```

Comment:
What about `legacy.version.org.wildfly.security.elytron-mp`? I'm not sure why its version is handled by a different maven property. A quick look reveals it is only used by `org.wildfly.security:wildfly-elytron-jwt`, which is a child module of `org.wildfly.security:wildfly-elytron-parent`


Otherwise, LGTM (from wildfly-core relathionship point of view)

Reply:
Thanks for catching that @yersan! I've updated this.